classes/Address.php

```diff
@@ -440,14 +440,23 @@ public static function getCountryAndState($id_address)
      *
      * @return bool The address exists
      */
-    public static function addressExists($id_address)
+    public static function addressExists($id_address, $refreshCache = false): bool
```

Comment:
```suggestion
    public static function addressExists($id_address, bool $refreshCache = false): bool
```

Could you add the PHPDoc ?